.github/workflows/update_semver.yml

```
@@ -0,0 +1,17 @@
+name: Update Semver
+on:
+  push:
+    tags:
+      - '*.*.*'
```

let's add the `v` prefix 😃 

```suggestion
      - 'v*.*.*'
```